docs/components/connectors/out-of-the-box-connectors/uipath.md

```diff
@@ -0,0 +1,243 @@
+      "x-xss-protection":"1",
+      "x-content-type-options":"nosniff",
+      "x-robots-tag":"noindex,nofollow",
+      "content-type":"application/json; odata.metadata=minimal; odata.streaming=true",
+      "location":"https://cloud.uipath.com/camunda1/DefaultTenant/orchestrator_/odata/QueueItems(436141352)",
+      "connection":"keep-alive",
+      "cache-control":"no-cache, no-store, must-revalidate",
+      "odata-version":"4.0"
+   },
+   "body":{
+      "@odata.context":"https://cloud.uipath.com/camunda1/DefaultTenant/orchestrator_/odata/$metadata#QueueItems/$entity",
+      "QueueDefinitionId":165001,
+      "Encrypted":false,
+      "Status":"New",
+      "ReviewStatus":"None",
+      "Key":"2196eb07-c96a-4f47-a734-326dd5d58a9d",
+      "Reference":"test",
+      "Priority":"Low",
+      "DeferDate":"2023-01-12T00:00:00Z",
+      "SecondsInPreviousAttempts":0,
+      "RetryNumber":0,
+      "SpecificData":"{\"DynamicProperties\":{\"test\":\"test\"}}",
+      "CreationTime":"2023-01-20T10:13:20.6603953Z",
+      "RowVersion":"AAAAAE2f4GY=",
+      "OrganizationUnitId":1964413,
+      "Id":436141352,
+      "SpecificContent":{
+         "test":"test"
+      }
+   }
+}
+```
+
+### Get Queue Item result by ID
+
+This operation allows you get the information related to a certain item from your UIPath Orchestrator. In order to execute it, you need to take the following steps:
+1. Select the operation 'Get Queue Item result by ID' from the dropdown list 'Operation type'.
+2. Configure authentication as described in section [Authentication](#Authentication)
+3. Fill response mapping as described in section [UIPath Connector response](#UIPath-Connector-response)
+
+![UIPath Connector - Get Queue Item result by ID](../img/connectors-uipath-get-queue-item-result-by-id.png)
+
+#### Get Queue Item result by ID response 
```

Comment:
we should add here, where the connector gets the id from?
prerequisite: you need to create a workitem with the connector before you can get the result

Reply:
Done